source/creating_doc/doc_templates.rst

```diff
@@ -230,11 +230,11 @@ Copy this codeblock to an iamge on its own line.
 
   Line of content, followed by a line with an image.
 
-  .. image:: _images/image-file-name
+  .. image:: /_images/image-file-name
```

Comment:
Any reason to not locate the images closer to the text that uses them?  Would we need to tell sphinx about every folder that has images?

Reply:
@feanil  i think that would end up with more duplications; in a single place it's easy to scan what exists and reuse